src/tests/test_pem.cpp

```diff
@@ -19,16 +20,31 @@ class PEM_Tests : public Test
          {
          Test::Result result("PEM encoding");
 
-         std::vector<uint8_t> vec = { 0, 1, 2, 3, 4 };
+         std::vector<uint8_t> vec1 = { 0, 1, 2, 3, 4 };
+         std::vector<uint8_t> vec2 = { 5, 6, 7, 8, 9 };
 
-         const std::string pem1 = Botan::PEM_Code::encode(vec, "BUNNY", 3);
+         const std::string pem1 = Botan::PEM_Code::encode(vec1, "BUNNY", 3);
+         const std::string pem2 = Botan::PEM_Code::encode(vec2, "BUNNY", 3);
+
+         const std::string pems = pem1 + "\n" + pem2;
```

Comment:
Frankly, I'm not sure whether this is allowed, but I think this should test something like that, too:
```
-----BEGIN BUNNY-----
AAE
CAw
Q=
-----END BUNNY-----

DUMB GARBAGE

-----BEGIN BUNNY-----
AAE
CAw
Q=
-----END BUNNY-----
```